doc/faq.mkd

```diff
@@ -50,3 +50,16 @@ on your remote repositories:
 [bitbucket-default-branch]: https://answers.atlassian.com/questions/280944/how-to-change-main-branch-in-bitbucket "How to change MAIN branch in BitBucket?"
 [gitolite-v2-default-branch]: http://stackoverflow.com/questions/7091599/git-default-remote-branch-with-gitolite "git default remote branch with gitolite"
 [gitolite-v3-default-branch]: http://stackoverflow.com/questions/13949093/git-change-default-branch-gitolite "git change default branch (gitolite)"
+
+#### I get the error: [R10K::Action::Deploy::Environment - ERROR] Couldn't update git cache for ...
```

Comment:
The problem is that this error means "a bad thing" happened, and either there's more information provided which can have specific remedying actions, or there's no more information so there's nothing to go on. Since this doesn't provide a lot of help in either case I would rather we omit this and fill in specific FAQ entries for specific errors.